Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update -q --fix-missing && \
     amavisd-new \
     arj \
     binutils \
+    bsd-mailx \
```

Comment:
Isn't sendmail working for this?

Reply:
I wasn't able to do so. But maybe it was just my fault.
Could you provide an example on how to use sendmail? Is there something already existing here I could adapt?

Reply:
https://github.com/tomav/docker-mailserver/blob/a73692cc9fb98849bb0eac72311f4966ecaaca41/test/tests.bats#L1368
I haven't tried using it, but the tests seem to work if one is using it like this.. 
I don't like the way sendmail works either. It's different on every system, sometimes a symlink to some other mail service and then there are about a couple different implementations, all with different syntaxes... :roll_eyes: